aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/kinesisvideo/http/KinesisVideoApacheHttpClient.java

```diff
@@ -86,13 +85,18 @@ public CloseableHttpResponse executeRequest() {
     }
 
     private CloseableHttpClient buildHttpClient() {
-        final SSLContextBuilder builder = new SSLContextBuilder();
         try {
-            builder.loadTrustMaterial(new TrustAllStrategy());
-            final SSLConnectionSocketFactory sslSocketFactory = new SSLConnectionSocketFactory(builder.build(),
-                    new String[] {"TLSv1.2"}, // TLS protocol, use 1.2 only
-                    null, // TLS ciphers, use default
-                    new NoOpHostNameVerifier());
+            final SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+            // Initializes context. Either of the first two parameters may be null in which case
+            // the installed security providers will be searched for the highest priority implementation
+            // of the appropriate factory. Likewise, the secure random parameter may be null in which case the default
+            // implementation will be used.
+            sslContext.init(null, null, null);
```

Comment:
Shouldn't you be initializing the `SSLContext` here the same way you do it in the `SocketFactory`?    As is, I don't think there will be any hostname verification for this client.

Reply:
There will still be a hostname verification as it's set a bit later, SSLIOSessionStrategy. I think either way is an okay approach. I have a manual test that verifies that this part works properly.

Reply:
I think the same, the second arg is the TrustManagers and specifying null for that seems like it would skip hostname verification.